.bazelrc

```diff
@@ -65,4 +65,4 @@ build:libc++ --define force_libcpp=enabled
 build:sizeopt -c opt --copt -Os
 
 # Test options
-test --test_env=HEAPCHECK=normal --test_env=PPROF_PATH
+build --test_env=HEAPCHECK=normal --test_env=PPROF_PATH
```

Comment:
Hmm, I don't think this change is needed?

Reply:
It is, otherwise `--test_env` changes between `bazel build` and `bazel test`.

Reply:
right, but that doesn't affect cache analysis.

Reply:
It does:
```
$ bazel --noworkspace_rc build //source/common/buffer:buffer_lib 2>&1 | grep test_env
$ bazel --noworkspace_rc test --test_env=TEST=1 //source/common/buffer:buffer_lib 2>&1 | grep test_env
INFO: Build option --test_env has changed, discarding analysis cache.
```
```
$ bazel --noworkspace_rc build --test_env=TEST=1 //source/common/buffer:buffer_lib 2>&1 | grep test_env
$ bazel --noworkspace_rc test --test_env=TEST=1 //source/common/buffer:buffer_lib 2>&1 | grep test_env
```

Reply:
With `.bazelrc` from `master` (which sets different `--test_env` flags for `bazel test`):
```
$ bazel build //source/common/buffer:buffer_lib 2>&1 | grep test_env
$ bazel test //source/common/buffer:buffer_lib 2>&1 | grep test_env
INFO: Build option --test_env has changed, discarding analysis cache.
```
After the change from this PR:
```
$ bazel build //source/common/buffer:buffer_lib 2>&1 | grep test_env
$ bazel test //source/common/buffer:buffer_lib 2>&1 | grep test_env
```

Reply:
Hmm, it seems bazel discard analysis cache while action cache is still valid so the build doesn't take longer time. If you feel strong about this change go for it. Can you merge master in that case?

Reply:
I think it makes sense to add it, otherwise this PR feels half-done. Merged master. Thanks!